jobs/payment-jobs/tasks/eft_task.py

```diff
@@ -449,3 +450,4 @@ def _handle_invoice_refund(cls, invoice: InvoiceModel, invoice_reference: Invoic
         invoice.refund_date = datetime.now(tz=timezone.utc)
         invoice.refund = invoice.total
         invoice.flush()
+        EftService().release_payment_or_reversal(invoice, TransactionStatus.REVERSED.value)
```

Comment:
Probably need something similar to pay-queue

Reply:
Also do we send a reversal when we put an EFT invoice back to APPROVED? I think so